src/Task/OnCompletion.ts

```diff
@@ -36,7 +36,7 @@ export function handleOnCompletion(originalTask: Task, newTasks: Task[]): Task[]
 
     const ocAction: OnCompletion = originalTask.onCompletion;
 
-    if (OnCompletion.Delete === ocAction) {
+    if (ocAction === OnCompletion.Delete) {
```

Comment:
Trivia: I accidentally pushed this change to `main` _after_ creating this PR.

So I expect/hope that when I merge this PR, this change will disappear from the PR...

Reply:
It didn't. But the graph is fine:

![image](https://github.com/user-attachments/assets/5714aa84-dac7-4788-bb1f-8a06c9af82b9)